saleor/graphql/shipping/dataloaders.py

```diff
@@ -111,6 +111,24 @@ def batch_load(self, keys):
         ]
 
 
+class ShippingMethodChannelListingByChannelSlugLoader(DataLoader):
+    context_key = "shippingmethodchannellisting_by_channel"
+
+    def batch_load(self, keys):
+        shipping_method_channel_listings = ShippingMethodChannelListing.objects.filter(
```

Comment:
We should add `.using(self.database_connection_name)` for all db queries in dataloaders. I sill working on docs for this database read replicas. 
```suggestion
        shipping_method_channel_listings = ShippingMethodChannelListing.objects.using(
            self.database_connection_name
        ).filter(
```